app/src/app/user/profile/profile.tpl.html

```diff
@@ -93,15 +93,83 @@
             <h3>Preferences</h3>
             <md-whiteframe class="md-whiteframe-z1" layout="column" layout-padding>
                 <md-select-label>TIMEZONE</md-select-label>
-                <md-select ng-model="ProfileController.user.timezoneIdentifier" aria-label="Timezone">
+                <md-select ng-model="ProfileController.fullUserInfo.timezoneIdentifier" aria-label="Timezone">
                     <md-option ng-value="timezone.timezoneIdentifier" ng-repeat="timezone in ProfileController.timezones">{{ timezone.timezoneIdentifier }}</md-option>
                 </md-select>
             </md-whiteframe>
         </md-content>
 
+        <md-content layout-padding>
+            <h3>Access</h3>
+            <md-whiteframe class="md-whiteframe-z1" layout="column" layout-padding>
+                <label>LOGIN METHODS</label>
+                <md-list>
+                    <md-list-item class="md-2-line">
+                        <md-icon>accessibility</md-icon>
+                        <div class="md-list-item-text">
+                            <h3>Facebook</h3>
+                            <h4>{{ ProfileController.fullUserInfo.hasFacebookLogin() ? 'Connected as: ' + ProfileController.fullUserInfo.fullName() : 'Not connected' }}</h4>
```

Comment:
It is better in these cases to just have two buttons - each with an ng-if. Ternary logic in views should be avoided.